src/sentry/static/sentry/app/components/forms/searchInput.tsx

```diff
@@ -56,9 +57,10 @@ const SearchReset = styled(props => <InlineSvg src="icon-circle-close" {...props
   }
 `;
 
-const SearchWrapper = styled('div')<{smaller: boolean}>`
+const SearchWrapper = styled('div')<{smaller: boolean; width: string}>`
   position: relative;
   display: inline-block;
+  width: ${p => (p.width ? p.width : 'auto')};
```

Comment:
@NisanthanNanthakumar I would prefer it if we didn't modify the search input for this PR which is not supposed to cover search functionality